src/libs/shim-proto/build.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2021 Alibaba Cloud
+// Copyright (c) 2021 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#[cfg(not(feature = "generate"))]
+fn main() {}
```

Comment:
panic!("Not supported"); ?
When this feature option will be used?

Reply:
> panic!("Not supported"); ? When this feature option will be used?

There are two possible ways to deal with auto-generated code:
1) do not publish the auto-generated code and generate them on demand.
2) publish the auto-generated code, and only regenerate the code when needed.
Here we adopt method 2,  feature "generate" is used to regenerate code. When feature "generate" is not enabled.